rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/stop_scan/action.py

```diff
@@ -17,20 +17,14 @@ def __init__(self):
     def run(self, params={}):
         scan_id = params.get(Input.ID)
         url = f"scan/{scan_id}/stop"
-        response = self.connection.ivm_cloud_api.call_api(url, "POST")
-        if response[0] == 202 and response[1] is None:
-            response = self.connection.ivm_cloud_api.call_api("scan/" + scan_id, "GET")
-            if response[1].get("status") == "Stopped":
-                return {Output.SUCCESS: True}
-            else:
-                return {
-                    Output.SUCCESS: False,
-                    Output.STATUS_CODE: 400,
-                    Output.MESSAGE: f"Failed to stop scan with ID '{scan_id}'. Status of scan is {response[1].get('status')}",
-                }
+        self.connection.ivm_cloud_api.call_api(url, "POST")
+
```

Comment:
Is this operation always instant? Do we need a sleep() here or something of that ilk to let the product catch up?

Reply:
I'm honestly not sure if this is instant or not

Reply:
This would be a synchronous call right? So it should only return once it's complete.

Reply:
Yeah, as far as I can tell looking at the API documentation and poking around, it only returns when it's done

Reply:
It's two calls though. You're calling once to do the action, then you're immediately calling again to see the status of that action. In my experience, it's a crap shoot (by product) on if the server is ready for that second call.

Reply:
That makes sense, I'll add in a sleep to be safe

Reply:
@aholland-r7 can you verify this with someone from the IVM API team? A 5 second wait to be safe isn't the end of the world, but if it's not necessary then we shouldn't implement it.